org.eclipse.scout.rt.rest.jersey.server/src/main/java/org/eclipse/scout/rt/rest/jersey/server/chunked/ChunkedDataWriter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.rest.jersey.server.chunked;
+
+import static org.eclipse.scout.rt.platform.util.Assertions.*;
+
+import java.io.IOException;
+
+import jakarta.ws.rs.core.GenericType;
+
+import org.eclipse.scout.rt.platform.util.StringUtility;
+import org.eclipse.scout.rt.rest.chunked.IChunkedDataWriter;
+import org.glassfish.jersey.server.ChunkedOutput;
+
+/**
+ * Scout REST API wrapper around Jersey {@link ChunkedOutput} instance.
+ * <p>
+ * See documentation
+ * <a href="https://eclipse-ee4j.github.io/jersey.github.io/documentation/latest/user-guide.html#chunked-output">Jersey chunked output</a>
+ */
+public class ChunkedDataWriter<T> implements IChunkedDataWriter<T> {
+
+  protected ChunkedOutput<T> m_output;
+
+  @Override
+  public void init(Class<T> clazz, String delimiter) {
+    assertNull(m_output, "already initialized");
+    assertNotNull(delimiter, "missing delimiter");
+
+    // Use delimiter specified by org.glassfish.jersey.client.ChunkedInput.parser (e.g. "\r\n") if no custom delimiter was specified.
+    // Providing a null or empty delimiter to ChunkedOutput leads to no delimiter at all which cannot be parsed by ChunkedInput later on.
+    // See https://stackoverflow.com/questions/40429196/chunkedoutput-response-from-jersey-rest-services
+    m_output = new ChunkedOutput<>(clazz, StringUtility.isNullOrEmpty(delimiter) ? "\r\n" : delimiter);
```

Comment:
Again, the use of StringUtility.isNullOrEmpty() seems superfluous with the previous assertion.